opencti-platform/opencti-graphql/config/schema/opencti.graphql

```diff
@@ -80,6 +80,18 @@ enum FilterMode {
   or
 }
 
+input Filter {
+  key: String
+  values: [String]
+  operator: String
+  mode: String
+}
+
+type FilterRepresentative {
```

Comment:
this will also be changed after the meeting with Julien yesterday
We want the back to return the type below when we ask for filterRepresentatives (query to fetch the values of the filters ids) --> open to discussion for the filterRepresentatives query returned type
![image](https://github.com/OpenCTI-Platform/opencti/assets/75783086/2dccfedf-5428-48b0-85c7-33df5c1283dd)